_posts/2017-07-06-React-Native-for-iOS-devs.markdown

```diff
@@ -61,8 +61,6 @@ To get a sense of what this feels like, I've created a simplified version of the
 <p>Warning: You may need to expand the width of this window to see the full diagram.</p>
 </div>
 
-</article>
```

Comment:
Unexpected HTML tags like these now cause the template to break and the markup to be rendered raw.